binary.json

```diff
@@ -46,14 +46,24 @@
          "expected": 31
       },
       {
-         "description": "numbers other than one and zero raise an error",
-         "binary": ["012", "2"],
-         "expected": -1
+        "description": "2 is not a valid binary digit and raises an error",
```

Comment:
I'd remove 'and raises an error' from the descriptions.
The error raising is an extension added by the Ruby track.

The readme just says "The program should handle invalid inputs."